spec/ruby/core/kernel/caller_spec.rb

```diff
@@ -11,7 +11,7 @@
   end
 
   it 'returns an Array of caller locations using a custom offset' do
-    locations = KernelSpecs::CallerTest.locations(2)
+    locations = KernelSpecs::CallerTest.locations(3)
```

Comment:
This commit (cb518a7058463ddf8923e4c329ce1d3c42710338) should be renamed.
Also, is the change still needed?

Reply:
hm I've renamed it, but it got lost somewhere. I needed it until rebase. Not true any more. I leave it up to you if it should be kept.